capi/geos_c.h.in

```diff
@@ -3635,6 +3658,69 @@ extern GEOSGeometry GEOS_DLL *GEOSOffsetCurve(const GEOSGeometry* g,
 
 ///@}
 
+
+/* ====================================================================== */
+/** @name Coverages
+* Functions to work with coverages represented by lists of polygons
+* that exactly share edge geometry.
+*/
+///@{
+
+/**
+* Analyze a coverage (represented as a collection of polygonal geometry
+* with exactly matching edge geometry) to find places where the
+* assumption of exactly matching edges is not met.
+*
+* \param input The polygonal coverage to access,
+*        stored in a geometry collection.
+* \return A value of 1 for a valid coverage, 0 for invalid and -1 for
+*         a processing error.
+*
+* \since 3.12
+*/
+extern int GEOSCoverageIsValid(
+    const GEOSGeometry* input);
+
+/**
+* Analyze a coverage (represented as a collection of polygonal geometry
+* with exactly matching edge geometry) to find places where the
+* assumption of exactly matching edges is not met.
+*
+* \param input The polygonal coverage to access,
+*        stored in a geometry collection.
+* \param gapWidth The maximum width of gaps to detect.
+* \return A collection containing a MULTILINESTRING indicating the
+*         invalid edges for each input polygon, or an empty
+*         MULTILINESTRING for polygons that are valid.
+*
+* \since 3.12
+*/
+extern GEOSGeometry GEOS_DLL *GEOSCoverageValid(
```

Comment:
Note that it's inefficient to call both `GEOSCoverageIsValid` and `GEOSCoverageValid` (`GEOSCoverageInvalidEdges`?), since that requires two full validation computations.  Just the nature of a function-based API.  There may be use cases for both, however.

Reply:
Done, integrated per Dan's suggestion.